### ISSUE TITLE
Promote --fix when standard detects issues

### ISSUE THREAD

**Issue by timoxley** [feross/standard#576]
#### Problem

`standard --fix` does a lot of hard work for the user and should reduce instances of linter-rage when faced with sweeping changes, such as the new JSX single -> double quote rule: https://github.com/feross/eslint-config-standard/issues/27.

However, `standard --fix` is new and many users will not know to even look for it. Some recent anecdotal evidence:

[![image](https://cloud.githubusercontent.com/assets/43438/17127998/30c2eb18-533c-11e6-81fc-c408853f768b.png)](https://github.com/feross/standard/issues/564#issuecomment-235172561)

This new & handy feature could do with more promotion, especially when it's most applicable: when the user is staring at a wall of linter errors.

#### Suggestion

Mention `--fix` whenever standard reports an error.

Issue stemming from this comment: https://github.com/feross/standard/issues/564#issuecomment-235167750

----

I'd submit a PR but it's not obvious what the wording should be or how this should be achieved.

Ideally the error message would span multiple lines like:

```
standard: Use JavaScript Standard Style (http://standardjs.com)
Try `standard --fix` to automatically fix problems.
```
But that's probably not possible in current setup as is. The [tagline](https://github.com/feross/standard/blob/4414c12449f4c1ce22b7fabe8e0bdf5950723860/options.js#L14) is printed in between `standard: ` and the `(homepage)` in the following:

```
standard: Use JavaScript Standard Style (http://standardjs.com)
```
Without looking too deeply it looks like the formatting of the error message is applied by eslint. Perhaps there are other options to get it to print a different thing?

Embedding it on the one line as a part of the tagline looks like:
```
standard: Use JavaScript Standard Style. Try `standard --fix` to automatically fix problems. (http://standardjs.com)
```

This is maybe just acceptable in this form, though it is a bit wordy. 

The tagline is not really the right place for this as it also infects `--help` (and perhaps other places?):

```
> standard --help
standard - Use JavaScript Standard Style. Try `standard --fix` to automatically fix problems. (http://standardjs.com)

Usage:
    standard <flags> [FILES...]

    If FILES is omitted, then all JavaScript source files (*.js, *.jsx) in the current
    working directory are checked, recursively.

    Certain paths (node_modules/, .git/, coverage/, *.min.js, bundle.js, vendor/) are
    automatically ignored.

Flags:
…
```

Maybe this is ok though?

Overall: `¯\_(ツ)_/¯ `

**Comment by feross** [comment]
Great idea -- this is easy to add in `standard-engine` which is where the support for `--fix` was added. PR here: https://github.com/Flet/standard-engine/pull/115
